Berksfile

```diff
@@ -1,3 +1,4 @@
 source "https://supermarket.chef.io"
 
 metadata
+cookbook 'socrata_configuration', git: 'git@git.socrata.com:socrata_configuration-cookbook.git'
```

Comment:
Yeah, that's a good question. I'd like to clean this book and make it more accessible to the community, but ultimately, there's no documentation at all on how to automate a metaflows box, so I want to work openly to create any kind of resource. Also I've had to collaborate a bit with the Metaflows support team, and having the book open helped with that.